tools/apidiff/report/markdown.go

```diff
@@ -141,7 +141,7 @@ func (t *MarkdownTable) writeHeader() {
 	}
 	t.sb.WriteString("| ")
 	t.sb.WriteString(strings.Join(t.headers, " | "))
-	t.sb.WriteString(" |\n")
+	t.sb.WriteString(" |")
```

Comment:
What was the problem here?

Reply:
These changes would make the table.String() to produce a table without a linebreak at the end of string. You could regard this line break was moved to the next line's beginning.

In the MarkdownWriter, we only have a `WriteLine` method, which will append a line break at the end of string, therefore to keep consistency, I would like the `table.String()` does not have a line break at the end.